app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = I18n.t("base_title")
+    if page_title.blank?
```

Comment:
viết gọn lại theo toán tử ba ngôi cũng được nhé
page_title.blank? ? base_title : "#{page_title} | #{base_title}"